### DISCUSSION TITLE
Why the URL has two slashes

### DISCUSSION THREAD

**Discussion by MrCharteredCurator** [community/community#23558] [Other Feature Feedback, Questions, & Ideas]
<aside class="onebox allowlistedgeneric" data-onebox-src="https://mrcharteredcurator.github.io//docs/configuration/">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/90040719/181077889-d85862c8-454b-4b5f-9c6b-020ad03541ac.png" class="site-icon" width="400" height="400">

      <a href="https://mrcharteredcurator.github.io//docs/configuration/" target="_blank" rel="noopener nofollow ugc">Mr Chartered Curator</a>
  </header>

  <article class="onebox-body">
    

<h3><a href="https://mrcharteredcurator.github.io//docs/configuration/" target="_blank" rel="noopener nofollow ugc">Configuration</a></h3>

  <p>Curating a No-nonsense knowledge-base for CA Students.</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>Can anybody look into it and help how to solve this?</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/MrCharteredCurator/mrcharteredcurator.github.io">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/MrCharteredCurator/mrcharteredcurator.github.io" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/90040719/181077891-51fce3db-efd5-42b5-b2ec-e2ddfb953c78.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/5/8/5804395ed6cf5f822058b339612f040f03b1d767_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/5/8/5804395ed6cf5f822058b339612f040f03b1d767_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/5/8/5804395ed6cf5f822058b339612f040f03b1d767.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/8/5804395ed6cf5f822058b339612f040f03b1d767_2_10x10.png"></div>

<h3><a href="https://github.com/MrCharteredCurator/mrcharteredcurator.github.io" target="_blank" rel="noopener nofollow ugc">GitHub - MrCharteredCurator/mrcharteredcurator.github.io:...</a></h3>

  <p>mrcharteredcurator.github.io 🚧 under setup 🚧. Contribute to MrCharteredCurator/mrcharteredcurator.github.io development by creating an account on GitHub.</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by jfullstackdev** [comment]
<p>because at the source page, it’s the same link with double // , but will still work if one, so simply update those links</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/9/79b551c9979c6f57fcde4ee56c6de45d4a1ef4af.png" data-download-href="/uploads/short-url/hmGfrD79WjnU7NmsxGAn1adGHFl.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181077893-fc5da36f-d051-48ee-815c-515d02400ac0.png" alt="image" data-base62-sha1="hmGfrD79WjnU7NmsxGAn1adGHFl" width="690" height="52" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/9/79b551c9979c6f57fcde4ee56c6de45d4a1ef4af_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1319×101 17.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment]
<p>take note also, you have a build error, the latest one,</p>

**Comment by MrCharteredCurator** [comment]
<p><a class="mention" href="/jdevstatic">@jdevstatic</a> Thanks for informing. The file is intact. I don’t understand why the build failed.</p>
<p>Before this the source for pages was set at root folder. I thought setting it to docs folder will set the wierd URL right.</p>
<p>I will revert changes and check back</p>

**Comment by MrCharteredCurator** [comment, marked as answer]
<p><a class="mention" href="/jdevstatic">@jdevstatic</a> Thanks for your support.<br>
I could solve the double slash problem by correcting the URL I fed in the <code>_config.yml</code> file. So to those coming afterwards, Input the url without the ending slash i.e. not <code>http://example.com/</code> but <code>http://example.com</code></p>
<p>And the build I think succeeded after turning back source to (root) folder in github pages settings.</p>
